src/pages/scientificServices/cli-auth/CliAuth.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import colors from 'src/libs/colors';
+
+export const CliAuth = () => {
+  const hash = window.location.hash;
+  const queryString = hash.includes('?') ? hash.split('?')[1] : '';
+  const queryParams = new URLSearchParams(queryString);
+  const authCode = queryParams.get('code');
```

Comment:
todo: confirm that this param does not get sent to mixpanel. I'm pretty sure that this page gets sent to Mixpanel as "pipelines-cli-auth" without any URL params, but i need to double check prior to merging